examples/panel-datalinks/src/components/DataLinksPanel.tsx

```diff
@@ -76,3 +78,14 @@ const getStyles = () => {
     `,
   };
 };
+
+// This one is created to support versions prior to 9.1.0
```

Comment:
Sorry, I am not sure I fully understand - is it because `config` was part / not part of the props of `DataLinksContextMenu` previously?

Reply:
It is required in 9.0 but dropped in 9.1

Reply:
I see, thanks 🙏

Reply:
So maybe we need to improve that comment?

Reply:
Hmm, yeah, probably that would make it easier to understand. Maybe something like the following?
```suggestion
// The `config` prop was required in <9.1.0 versions but got dropped in 9.1.0. This type is here to support both Grafana version ranges.
```